### PR TITLE
Completed Homework 02

### PR DESCRIPTION
**Pull request by rickymetz**

### CODE REVIEW

**Comment by rickymetz** [comment]
Comfort : 5

File: .gitconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[user]
+    name = Rick Metzger
+    email = rickymetz@gmail.com
```
Comment (mobjohnson) [inline .gitconfig]: Rick, this file is outside of your student folder and should not be included in a pull request. Please let me know if you need further help with this.

File: class02/rick/air_conditioning.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	end
+elsif ac_status == "no"
+	if current_temp == desired_temp
+		puts "Your A/C is broke, but it's pretty comfortable in here."
+	elsif current_temp > desired_temp
+		puts "Fix the A/C now!  It's hot!"
+	else 
+		puts "Fix the A/C whenever you have the chance...  It's cool..."
+	end	
+end
```
Comment (mobjohnson) [inline class02/rick/air_conditioning.rb]: This looks good.

File: class02/rick/pyramid.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	puts "Invalid input. Printing Righside up pyramid."
+	#rightside up
+	character_count = 1
+	user_row_number.downto(1) do | space |
+		space -= 1
+		puts  " " * space + (user_character * character_count)
+		character_count += 2
+	end
+end
+
```
Comment (mobjohnson) [inline class02/rick/pyramid.rb]: Nice error handling on user input!

**Pull request closed without merging**